src/System.Net.Security/src/System/Net/Security/SslConnectionInfo.OSX.cs

```diff
@@ -60,85 +91,797 @@ private struct TlsMapping
             internal HashAlgorithmType HashAlgorithm;
             internal int HashAlgorithmStrength;
 
-            internal static TlsMapping EcDhe(CipherAlgorithmType cipher, HashAlgorithmType hash)
+            public static TlsMapping DhEphem(CipherAlgorithmType cipher, HashAlgorithmType hash)
             {
-                int cipherSize;
-                int hashSize;
+                return Build(ExchangeAlgorithmType.DiffieHellman, cipher, GetCipherSize(cipher), hash);
+            }
```

Comment:
Nit: This all seem like a great time to use expression-bodied members to make it a bit more concise, e.g.
```C#
public static TlsMapping DhEphem(CipherAlgorithmType cipher, HashAlgorithm hash) =>
    Build(ExchangeAlgorithmType.DiffieHellman, cipher, GetCipherSize(cipher));
```